src/store/foodprint-context.tsx

```diff
@@ -168,6 +179,15 @@ const FoodprintContextProvider: React.FC = (props) => {
     setFavorites(array);
   };
 
+  const searchResultResetHandler = () => {
+    return setRecipeSearchResults([IngredientSearchDefault]);
+  }
+
+  const addSearchResultsHandler = (response: IngredientSearch[]) => {
+    setRecipeSearchResults(response)
+  }
+
```

Comment:
whats the purpose of these 2? they look extremely similar -- but one returns and the other doesnt return

Reply:
why do we need a addSearchResults func, and why not just call setRecipeSearchResults?

Reply:
The purpose of the first is to reset the state to default. It is not currently serving any purpose and can be removed.

The second handler is to set the state in a ts file that's not a functional component (spoonacular requests). I can try passing in the setter directly again.